pages/common/meson.md

```diff
@@ -0,0 +1,24 @@
+# meson
+
+> SCons-like build system that uses python as a front-end language and Ninja as a building backend.
+> More information: <https://mesonbuild.com>.
+
+- Generate a c project with a given name and version:
+
+`meson init --language={{c}} --name={{myproject}} --version={{0.1}}`
+
+- Configure builddir with default values:
+
+`meson setup {{builddir}}`
```

Comment:
I have a question: Should the user run this command in the working directory where the `meson.build` file is located and `builddir` is the convention recommended name?

Reply:
the user  can use `-C` specify directory where `meson.build` located, question 2 is yes.

Reply:
because, based on our [style-guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md), we recommend the use of `{{path/to/builddir}}` but there are exceptions to the rule and maybe this the case. What do you think @sbrl?

Reply:
@navarroaxel  sorry, I have made some mistatakes in previous reply. `meson.build` must locate in working directory ,`-C` is to specify build directory

Reply:
Hrm. I think `{{path/to/build_dir}}` might be better.
```suggestion
`meson setup {{build_dir}}`
```